Configuration/AlCa/python/autoCond.py

```diff
@@ -82,6 +82,20 @@
     'phase2_realistic'         : '110X_mcRun4_realistic_v3'
 }
 
+##
+## Append for 0T conditions
+##
+import six
+from Configuration.StandardSequences.CondDBESSource_cff import GlobalTag as essource
+connectionString = essource.connect.value()
+ConditionsFor0T =  ','.join( ['RunInfo_0T_v1_mc', "RunInfoRcd", connectionString, "", "2020-07-01 12:00:00.000"] )
+GlobalTags0T = {}
+for key,val in six.iteritems(autoCond):
+    if "phase" in key:
+        GlobalTags0T[key+"_0T"] = (autoCond[key], ConditionsFor0T)
+
+autoCond.update(GlobalTags0T)
+
```

Comment:
Is there a reason that you have included this snippet here rather than in a separate file with a name such as `autoCondSpecial.py` or `autoCondModifiers.py`, analogous to the treatment of `autoCondHLT` and `autoCondPhase2` below? Once the additional modifiers to convert the Run-2 HEfail and cosmic GTs to symbolic GTs, they would also be contained in the same file.

Reply:
no particular reason, can be move elsewhere if desired. It didn't know about additional use cases and didn't want to clutter the folder with one more 10 lines file.

Reply:
@christopheralanwest I followed your suggestion and went ahead to created `Configuration/AlCa/python/autoCondModifiers.py`. Is that what you had in mind?

Reply:
Thanks, that's exactly what I had in mind.